kaldb/src/main/java/com/slack/kaldb/logstore/search/SearchQuery.java

```diff
@@ -10,20 +12,23 @@ public class SearchQuery {
   public final long endTimeEpochMs;
   public final int howMany;
   public final int bucketCount;
+  public final List<String> chunkIds;
 
   public SearchQuery(
       String dataset,
       String queryStr,
       long startTimeEpochMs,
       long endTimeEpochMs,
       int howMany,
-      int bucketCount) {
+      int bucketCount,
+      List<String> chunkIds) {
```

Comment:
I propose we add another ctor which takes the additional `chunkIds` . That will make the PR significantly smaller

Reply:
Leaving it as is since it's better to be explicit in the call params.